.eslintrc

```diff
@@ -1,6 +1,10 @@
 {
   "parser": "babel-eslint",
-  "extends": "airbnb",
+  "extends": [
+    "airbnb",
+    "prettier",
+    "prettier/flowtype"
+  ],
   "plugins": ["flowtype", "flowtype-errors", "prettier"],
```

Comment:
IIRC, we can ditch prettier plugin (which runs it as eslint rule) if we use prettier as standalone?

Reply:
That's correct, but it's preferred to run it as part of `eslint` per @philpl's comment [here](https://github.com/styled-components/styled-components/pull/1376#discussion_r157919895).